docs/Course Wiki/ECE Course Offerings/ECE310.md

```diff
@@ -0,0 +1,79 @@
+# ECE310
+
+ECE310 (Digital Signal Processing 1) is a 3-credit-hour course that satisfies the 1-of-6 Electrical Engineering Foundations Course requirement for CEs and satisfies one of the 3-of-5 Advanced Core Electives for EEs. It is offered in the fall, spring, and summer.
+
+## Content Covered
+
+- Discrete-time systems
+- Linear and time-invariant systems
+- Discrete convolution
+- Z-transform
+- Discrete-time fourier transform (DTFT)
+- Fourier analysis of LTI systems
+- Ideal filters
+- Sampling of continuous-time signals
+- Ideal Analog to Digital (A/D) and Digital to Analog (D/A) conversion
+- Discrete Fourier Transform (DFT)
+- Spectral analysis
+- Fast Fourier Transform
+- Downsampling and Upsampling
+- Finite Impulse Response (FIR) and Infinite Impulse Response (IIR) filters
+- Practical A/D and D/A conversion
+
+
+
+The first part of ECE310 is a brief review of continuous time signals, complex number arithmetic, and linear time-invariant causal (LTIC) systems. There is an emphasis on evaluating whether or not a system is LTIC. You are also introduced to digital signals and systems as well as discrete convolution. The next part of the course covers the Z-transform. Review partial fraction decomposition and polynomial long division. The course continues by talking about how to analyze systems via the Z-transform, how to evaluate the stability of said systems, as well as applications of LTIC systems. The course goes on to introduce a plethora of digital analogues to continuous Fourier transforms such as the Discrete-time Fourier transform (DTFT), Discrete Fourier transform (DFT), and the Fast Fourier transform (FFT). Interwoven into all of these new transforms is the concept of Analog to Digital and Digital to Analog conversion which reappears throughout the course. The end of the class delves deeper into digital filter design, multirate signal processing, practical D/A and A/D converter design, and applications of digital signal processing. The course places great emphasis on mathematical prowess and in depth knowledge of the content.
+
+This class does not require [ECE311](ECE311.md) (the lab section) but it is highly recommended to take it concurrently with ECE310. It is also recommended to take ECE310 soon after ECE210 as the courses do have some overlap as it pertains to transforms and system properties. Do not fret! Doing well in ECE210 is not required to do well in ECE310. 
```

Comment:
"as the courses do have some overlap as it pertains" -> "as the courses do have some overlap in concepts, such as transforms and ..."